enclave/server.ts

```diff
@@ -51,11 +51,57 @@ const nStates = new ethers.Contract(
     signer
 );
 
+type ClaimedMoved = {
+    socketId: string;
```

Comment:
niceee i like the type for this 

why do we need socketId? can't we keep a map of public keys to sockets? less surface area for attack + helps us with sybil protection